src/lib/slide-toggle/slide-toggle.ts

```diff
@@ -142,7 +159,8 @@ export class MatSlideToggle extends _MatSlideToggleMixinBase implements OnDestro
   @ViewChild(MatRipple) _ripple: MatRipple;
 
   constructor(elementRef: ElementRef,
-              private _platform: Platform,
+              // @deletion-target 7.0.0 _platform parameter no longer necessary.
```

Comment:
Should that be a multi line comment, so that it will be preserved, and maybe displayed in Dgeni?